docs/tutorials/private-network/create-qbft-network.md

```diff
@@ -159,11 +159,11 @@ Update the IP and port numbers for all initial validator nodes in `static-nodes.
 Copy `static-nodes.json`, `genesis.json`, and `permissioned-nodes.json` (if applicable) to the data directory for each node:
 
 ```bash
-cp static-nodes.json genesis.json ./../Node-0/data/
-cp static-nodes.json genesis.json ./../Node-1/data/
-cp static-nodes.json genesis.json ./../Node-2/data/
-cp static-nodes.json genesis.json ./../Node-3/data/
-cp static-nodes.json genesis.json ./../Node-4/data/
+cp static-nodes.json genesis.json ../../Node-0/data/
+cp static-nodes.json genesis.json ../../Node-1/data/
+cp static-nodes.json genesis.json ../../Node-2/data/
+cp static-nodes.json genesis.json ../../Node-3/data/
+cp static-nodes.json genesis.json ../../Node-4/data/
```

Comment:
Should we have Node-5 added here?

Reply:
No, this is the tutorial to create the base QBFT network (with nodes 0-4). The other changes in this PR are for the separate tutorial to add Node 5.